intranet/static/js/bus.js

```diff
@@ -1,707 +1,715 @@
 /* globals Messenger */
 var bus = {};
+var now = new Date();
+afterNoon = now.getHours() > parseInt("{{ changeover_time }}", 10);
 
 $(function() {
```

Comment:
There's another problem with this JS as it stands: Some users are probably going to open `/bus` in the morning, leave it open until the afternoon, then pull the page back up and expect it to work without refreshing. I haven't tested the UI yet, but I doubt it handles that case. And this could become important.

Here's what I would recommend: For the `home` view ONLY (this may require passing an extra variable through the `morning` and `afternoon` views), if it passes the changeover time (either way, morning -> afternoon or afternoon -> morning), then the page should display a popup that tells the user they need to refresh to display accurate results.

Reply:
I force a refresh on signage, what about doing that here?

Reply:
I would rather leave it up to the user. @theo-o?